numpy/typing/__init__.py

```diff
@@ -156,13 +156,14 @@
 # further down in this file
 
 from numpy._typing import (
+    Array,
     ArrayLike,
     DTypeLike,
     NBitBase,
     NDArray,
 )
 
-__all__ = ["ArrayLike", "DTypeLike", "NBitBase", "NDArray"]
+__all__ = ["Array", "ArrayLike", "DTypeLike", "NBitBase", "NDArray"]
```

Comment:
It looks like introduced a new warning into the numpy doc build:

```
/home/circleci/repo/venv/lib/python3.11/site-packages/numpy/typing/__init__.py:docstring of numpy.typing:220: WARNING: Inline interpreted text or phrase reference start-string without end-string.
```

This is probably caused by the dynamic editing of the docstring below. I'm not sure why adding something now causes this but it was fine before.

Reply:
Thanks!  Yeah it looks like the docs don't build the link `` `npt.Array <numpy.typing.Array>` ``.  Scratching my head but I'm on it.